### ISSUE TITLE
hide 'The Internet'

### ISSUE THREAD

**Issue by errordeveloper** [weaveworks/scope#3358]
When all/most services make outbound connections, graph can become very hard to read.

Here is an example screenshot of what that may look like:

![image](https://user-images.githubusercontent.com/251467/46132469-f8eae900-c235-11e8-9c11-802ff6b71aa9.png)

**Comment by rade** [comment]
'hiding the Internet' is one possible solution but certainly not the only one. For example, we could render edges as just stubs when they go to/from highly connected nodes, and not take these edges into account for layout. Hovering over a stub could reveal the full edge.

**Comment by errordeveloper** [comment]
@rade I agree. The way I phrased this was based on what user has asked: "I want to hide The Internet (outbound connection)".

**Comment by dholbach** [comment]
Related to #1614?

**Comment by bboreham** [comment]
See also #1092 #1614
